### PR TITLE
AlienVault OTX v2 - handle non lower-case URLs and insecure err msg

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/40896
fixes: https://github.com/demisto/etc/issues/40905

## Description
-  handle no `status_code` attribute
- lowercase url as alienvault expects
- modified the tested ip cause private ip is not allowed

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

**Comment by Itay4** [comment]
TDD: see test failure at https://code.pan.run/xsoar/content/-/jobs/7368963

**Review by glicht — APPROVED** [review]
Please add a test for lowercase protocol fix

File: Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/README.md [hunk 1]
```diff
@@ -974,3 +974,6 @@ Queries a URL in AlienVault OTX.
 >|---|---|---|---|---|
 >| http://www.alexa.com/siteinfo/fotoidea.com | fotoidea.com | www.fotoidea.com | http://www.fotoidea.com/sport/4x4_san_ponso/slides/IMG_0068.html/url_list | http://whois.domaintools.com/fotoidea.com |
 
+
+## Additional Information
+ - AlienVault consider non lowercased URL protocol as invalid, and hence such URL protocols will be lowercased by the integration and returned as such.
```
Comment (bakatzir) [inline Packs/AlienVault_OTX/Integrations/AlienVault_OTX_v2/README.md:979]:
```suggestion
 - AlienVault considers non lowercased URL protocol as invalid, e.g, HTTP://www.google.com. Hence such submissions will be lowercased to ensure a seamless usage of the integration.
```

File: Packs/AlienVault_OTX/ReleaseNotes/1_1_8.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### AlienVault OTX v2
+- Fixed an issue where the ***url*** did not handle URLs with non lowercase protocol properly.
```
Comment (bakatzir) [inline Packs/AlienVault_OTX/ReleaseNotes/1_1_8.md:3]:
```suggestion
- Fixed an issue where the ***url*** command did not handle URLs with non lowercase protocol properly.
```

**Comment by Itay4** [comment]
build fails on non related test (misp v3)
merging

**Pull request merged**